pages/api/account/statistics.js

```diff
@@ -24,9 +24,16 @@ export default async function handler(req, res) {
   res.status(200).json(data);
 }
 
-export async function getStats(username) {
+export async function getStats(username, numberOfDays = 30) {
   await connectMongo();
 
+  // is the format okay? or should it be in this => '1m, 2m, 1y, 4m' instead of the numbers
```

Comment:
It was more of a question and a suggestion but I will remove the comments, what I meant by the question is if we better have the numverOfDays passed in to be "1d", "1m" instead of 30, "6m" instead of 180.

Reply:
I think for now we can keep it to days 👍